libs/ui-lib/lib/ocm/config/constants.ts

```diff
@@ -41,3 +41,4 @@ export const ocmPermissionsToAIPermissions = (
 };
 
 export const OCM_CLUSTER_LIST_LINK = '/openshift'; // TODO(mlibra): Tweak it!!!
+export const EXTERNAL_PARTNER_INTEGRATIONS_MIN_VERSION = '4.14.0';
```

Comment:
We should be basing the OCI control's state on the `EXTERNAL_PLATFORM_OCI` entry in the `/features` API response. Not a constant.

Reply:
True. I've missed this variable. I'll change it